web/src/components/hero.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Img from 'gatsby-image';
+
+export default function Hero({ image }) {
+    const {
+        childImageSharp: { fluid },
+    } = image;
+
+    return (
+        <div className="relative text-center text-white w-full">
+            <Img
+                fluid={fluid}
+                alt="A plane"
+                placeholder="blurred"
+                className="transform -translate-y-24 -z-2"
+            />
+            <h1 className="absolute left-2/4 top-2/4 transform -translate-y-2/4 -translate-x-2/4 text-ts-h1">
+                Donec eleifend faucibus finibus maximus
+            </h1>
+        </div>
```

Comment:
I wonder if using `gatsby-background-image` might simplify some of this layout. We used it for [Hubble](https://github.com/bckrm/hubble-technology/blob/main/web/src/components/index/indexHero.js#L59-L68). We could get away from some of the relative/absolute positioning and transform stuff by using flex.

What is the `-z-2` for? My guess is there's a conflict with the header? If that's the case that would probably be solved by adding `fixed` to the header.

If that's not the solve and you do need the negative z-index, we don't have to extend the base config like that anymore. You could just write `className="-z-[value]"` and tailwind will convert that to a class. There is some weirdness with how it calculates dynamic values (you have to store the value in a variable and then pass that variable in a template literal).